updated gitrepositories.md (fixes #341)

LordJashin32 opened the pull request:
### Description
Added link to Git GUI's, a Github fork 1 click sychronizer, and
added the pretense of "reward" for the long steps climbed so far for 
new aspiring VI's. 

### Raw.Githack preview link
https://raw.githack.com/LordJashin32/lordjashin32.github.io/tristans-best-branch-1/#!pages/vi/gitrepositories.md

LordJashin32 committed:
Added Git Gui Links Section

irisb1701 commented:
It's a nice addition, but probably deserves a tutorial under the blog section rather than get stashed at the bottom of step 5. I would maybe just add the git gui link underneath the useful links section for now, what do you think @dogi

LordJashin32 commented:
I'm going to have to change my mind on this a bit. It seems GitExtensions multi-platform is really bad. I think I'm going to just have to list what works best for each platform in particular. 
This Rabbit thing works just like tortoisegit: https://i.imgur.com/Re4Kdim.png .

LordJashin32 committed:
Update gitrepositories.md

found cross-platform apps unresponsive on Linux. Changing course to better results.

LordJashin32 committed:
Update gitrepositories.md

Iris b was right we need a bigger section to explain the importance of this.

LordJashin32 commented:
> 
> 
> It's a nice addition, but probably deserves a tutorial under the blog section rather than get stashed at the bottom of step 5. I would maybe just add the git gui link underneath the useful links section for now, what do you think @dogi

I think you're right it would be better to have a bigger section to explain the overarching importance of this in developer workflow. I'll leave the git gui links though because that's more general and not too specific like I was doing recommending specific programs. 
EDIT: Here's the final: https://raw.githack.com/LordJashin32/lordjashin32.github.io/1b36c451ab476e1a540c26565199fea6491266b8/#!pages/vi/gitrepositories.md

LordJashin32 committed:
Update gitrepositories.md

unabbreviated because there was room

irisb1701 commented on pages/vi/gitrepositories.md:
```diff
@@ -218,6 +218,7 @@ If you would like to understand how syncing with the fork works, here is a usefu
 [Git-it Workshop](http://jlord.us/git-it/) - Runs in your terminal to work and provides a hands-on approach to learn Git and GitHub repositories.
 [Git help](https://git-scm.com/) - An encyclopedia of useful git workflows and terminology explanations.
 [Other helpful links and videos](faq.md#Helpful_Links)
+[List of Git Graphical User Interfaces](https://git-scm.com/downloads/guis)
```
these two just need to be reversed

LordJashin32 committed:
Update gitrepositories.md

LordJashin32 committed:
Update gitrepositories.md

LordJashin32 commented:
Sorry, this PR took a while. Should rename this to tristan's worst branch. I've been sick like 2-3 days out of the past week it just won't stop raining here. 

I like these changes and I hope new VIs will see this link and use the GUIs available after they are done learning the command line. Or even while they learn it. It helped me a great amount. Multiple ways to do the same thing is good.

dogi committed:
Merge branch 'master' into tristans-best-branch-1

dogi committed:
Update gitrepositories.md
